resources/services/security/gen.hcl

```diff
@@ -0,0 +1,106 @@
+service = "gcp"
+output_directory = "."
+add_generate     = true
+
+resource "gcp" "security" "secrets" {
+  path = "google.golang.org/api/secretmanager/v1.Secret"
+
+  options {
+    primary_keys = [
+      "resource_name"
```

Comment:
wouldn't the project_id + id of the secret be more correct? also querying wise it will be faster (we don't add indexes)

Reply:
I think the idea was to be similar to 'arn' (which we always prefer for PK). 
Not sure what you mean about the speed of querying, but probably out of scope for this PR? (e.g. most AWS resources use 'arn' I think for PK...).

In any case, no hard beliefs either way, just reply again with which way you prefer :)

Reply:
But actually, yeah, don't think we need to worry too much about querying speed.. really not the bottleneck in cloudquery right now (bottleneck probably is 2h fetches or 12 hour fetches and so on)...

- And the 'id' column is a parsed column, so I kinda prefer the PK to be 'straight from the API'.